replication_report-extensions.Rmd

```diff
@@ -0,0 +1,361 @@
+dat_egood$music <- as.numeric(dat_egood$music)
+dat_egood <- dat_egood %>%
+  mutate(music.recode2 = (music - 1))
+
+#Set factor names:
+music_condition <- factor(dat_egood$music.recode2,
+                    levels = c(0, 1),
+                    labels = c("Liked", "DisLiked"))
+
+pen_choice <- factor(dat_egood$chose_advertised_pen,
+                levels = c(1, 0),
+                labels = c("ChoseAdPen", "NotChoseAdPen"))
+
+gender <- factor(dat_egood$geslacht1,
+                    levels = c(1,2),
+                    labels = c("Male", "Female"))
+
+# for relationship between advertised pen and music:
+fit1 <- glm(pen_choice ~ music_condition, data = dat_egood, family = binomial)
+fit1 <- tidy(fit1)
+fit1
+
+#### calculate odds 
+fit1$estimate[1] + fit1$estimate[2]
+
+## odds ratio
+exp(fit1$estimate[2])
+
+## predicted probabilities:
+# Disliked Music:
+(exp(fit1$estimate[1] + fit1$estimate[2])) / (1 + exp(fit1$estimate[1] + fit1$estimate[2]))
+# Liked Music: 
+exp(fit1$estimate[1]) / (1 + exp(fit1$estimate[1])) 
+
+# Does the relationship change when looking ONLY at males & ONLY females?
+
+male.dat_egood <- dat_egood %>%
+  filter(geslacht1 == "1")
+##double-check correct filtering:
+#count(male.dat_egood, geslacht1)
+
+female.dat_egood <- dat_egood %>%
+  filter(geslacht1 == "2")
+##double-check correct filtering:
+#count(female.dat_egood, geslacht1)
+
+# MALES:
+fitmale <- glm(pen_choice ~ music_condition, data = male.dat_egood, family=binomial)
+fitmale <- tidy(fitmale)
+fitmale
+
+#### calculate odds 
+fitmale$estimate[1] + fitmale$estimate[2]
+## odds ratio
+exp(fitmale$estimate[2])
+
+# FEMALES:
+fitfemale <- glm(pen_choice ~ music_condition, data = female.dat_egood, family=binomial)
+fitfemale <- tidy(fitfemale)
+fitfemale
+
+#### calculate odds 
+fitfemale$estimate[1] + fitfemale$estimate[2]
+## odds ratio
+exp(fitfemale$estimate[2])
+
+### predicted probabilities = confirmed, gender does not have a moderating effect on the likelihood 
+# of choosing the advertised pen when listening to liked versus disliked music.
+# Males - Disliked Music
+(exp(fitmale$estimate[1] + fitmale$estimate[2])) / 
+  (1 + exp(fitmale$estimate[1] + fitmale$estimate[2])) 
+# Males - Liked Music
+exp(fitmale$estimate[1]) / (1 + exp(fitmale$estimate[1])) 
+
+# Females - Disliked Music
+(exp(fitfemale$estimate[1] + fitfemale$estimate[2])) / 
+  (1 + exp(fitfemale$estimate[1] + fitfemale$estimate[2])) 
+# Females - Liked Music
+exp(fitfemale$estimate[1]) / (1 + exp(fitfemale$estimate[1])) 
+
+```
+The predicted probabilities for participants' pen choice in either the liked or disliked conditions did not change after splitting the sample by gender. Males and females were equally likely to choose the same pen, regardless of their assigned condition. However, these results should be qualified by the observation that this data contained over triple the number of female participants as male participants (see section 3.2.1).
```

Comment:
Overall, very good.

If I had to recommend one and only one possible improvement it would be more clear variable names and coding schemes.  That would help a lot with readability and would reduce the chance for errors.

Thanks.  I learned a lot from this example.